docs/src/main/sphinx/language/types.md

```diff
@@ -41,31 +41,58 @@ This type captures boolean values `true` and `false`.
 
 ## Integer
 
+Integer numbers can be expressed as numeric literals in the following formats:
+
+* Decimal integer. Examples are `-7`, `0`, or `3`.
+* Hexadecimal integer composed of `0X` or `0x` and the value. Examples are
+  `0x0A` for decimal `10` or `0x11` for decimal `17`.
+* Octal integer composed of `0O` or `0o` and the value. Examples are `0o40` for
+  decimal `32` or `0o11` for decimal `9`.
+* Binary integer composed of `0B` or `0b` and the value. Examples are `0b1001`
+  for decimal `9` or `0b101010` for decimal `42``.
+
+Underscore characters are ignored within literal values, and can be used to
+increase readability. For example, decimal integer `123_456.789_123` is
+equivalent to `123456.789123`. Preceding underscores are not permitted.
+Preceding underscores and trailing underscores are not permitted.
+
+Integers are supported by the following data types with increasing size range.
```

Comment:
What's "size range"? I would just remove the "with increasing size range clause". It doesn't add much and the language is confusing.

Reply:
Removed